model/component-feature.js

```diff
@@ -110,6 +111,27 @@ class ComponentFeature {
 
         return valid;
     }
+
+    /**
+     * Draw this Component Feature on the specified PaperScope.
+     *
+     * @param {PaperScope}  paperScope  The PaperScope object on which to draw
+     *                                  the Component Feature.
+     * @returns {paper.Path.Rectangle}  The bounding box that was just drawn represented as a
+     *                                  Path object.
+     */
+    print(paperScope) {
+        // First we need to create an "abstract" Rectangle object that will define our bounding box
+        let rect = new paper.Rectangle(0, 0, this.xSpan, this.ySpan);
+        rect.topLeft = new paper.Point(this.location.x, this.location.y);
+        // Next we have to draw the rectangle on the PaperScope project
+        let boundingBox = new paperScope.Path.Rectangle(rect);
+        boundingBox.fillColor = 'black';
```

Comment:
I can create a config file. Would you also like an option to change the color on the web page?

Reply:
I've created a config module. The plan is to write a Config parser and a schema so we can just upload a config.json or set the fields individually on the page.

Reply:
I don't think we need something custom since we really just need to set some variable values. There should be an existing library we can use that will read in a JSON or YAML file and build a dict of values we can reference in the JavaScript. We can worry about allowing the colors to be switched from the website later.

Reply:
Sure thing. We should just be able to parse a ```config.json``` file and store it as the config object, assuming it has all of the correct variable names.